quiz.py

```diff
@@ -1,6 +1,64 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Matthew Daguanno
+Date: 2024-03-04
+Description: Asks 3 questions and gives 3 options to pick from for each.
+"""
+
+print("Multiple Choice Quiz Game")
+
+counter = 0
+
+# Question one
+q1_inp = input("1. What is the capital of France?")
+print("(a) Paris")
+print("(b) London")
+print("(c) Rome")
+
+if q1_inp == "a":
+    print("Correct!")
+    counter = counter + 1
+elif q1_inp == "b":
+    print("Incorrect.")
+elif q1_inp == "c":
+    print("Incorrect.")
+else:
+    print(str(q1_inp) + " is not an option.")
```

Comment:
This is interesting, although in the context of a quiz game, it's a little unsatisfying because you are not able to re-answer the question. Do you know how to use loops yet? (It's the next chapter in the CodeHS notes.)